### ISSUE TITLE
Allow more customisation to toString formats - specifically dropping the currency code

### ISSUE THREAD

**Issue by astubbs** [JavaMoney/jsr354-ri#364]
I have a use case where i present many amounts all of the same currency on one table. I want to remove the currency code from the toString which is used by my templating framework. The ui will just indicate that the entire table is in one currency.

I've patched the RI to allow just "a" as the format string, which does what I need.

**Comment by keilw** [comment]
Would you be able to share this patch or PR here?

**Comment by astubbs** [comment]
Yes sure - but you won't like it ;), here it is. Allows you to specify just 'a' as the format:



Although it's probably faster to wrap Money in your own class, and use Lombok's `@Delegate` annotation and then override the toString formatter. That's what I ended up doing as there were other things I needed to change too.
